lib/Config/LocationAwareConfigRepository.php

```diff
@@ -52,12 +56,21 @@ public function loadConfigByKey(string $key): array
     {
         $dataSource = null;
 
-        // try to load from container config
-        $data = $this->getDataFromContainerConfig($key, $dataSource);
+        $loadType= $this->storageConfig['read_target']['type'];
```

Comment:
```suggestion
        $loadType= $this->storageConfig['read_target']['type'] ?? null;
```
This node is not available on all configs.